### PR TITLE
Chef course added in courses/free-courses-en.md

### PR DESCRIPTION
**Pull request by Manish-Syal123**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resource(s)
### Why is this valuable (or not)?
Chef is an ```open source systems management``` and ```cloud infrastructure automation platform```. Chef can manage a variety of node types, including servers, cloud virtual machines, network devices and containers. It manages Linux, Windows, mainframe and several other systems.
### How do we know it's really free?
It is available freely  on youtube
### For book lists, is it a book? For course lists, is it a course? etc.
It's a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -296,6 +297,11 @@
 * [Introduction to C++](https://www.edx.org/course/introduction-to-c-3) - Microsoft (edX) (:card_file_box: *archived*)
 
 
+###Chef
+
+* [CHEF Tutorial](https://www.youtube.com/playlist?list=PLsgnv1SN76ILtD3TnVtXpX1hmwjyY9OuT) - Online Tutorials
```
Comment (eshellman) [inline courses/free-courses-en.md:302]: isn't this devops?
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Comment (Manish-Syal123) [reply courses/free-courses-en.md:302]: May I add more free courses links for this course (chef) and add this course (chef) under Devops course ?

Comment (eshellman) [reply courses/free-courses-en.md:302]: yes

Comment (Manish-Syal123) [reply courses/free-courses-en.md:302]: I found only two free courses related to this.
There are many  chef courses on youtube but those are not in the form of playlist.

###Chef

* [CHEF Tutorial](https://www.youtube.com/playlist?list=PLsgnv1SN76ILtD3TnVtXpX1hmwjyY9OuT) - Online Tutorials
* [Learning Chef](https://www.youtube.com/playlist?list=PLKK5zTDXqzFM53J6-rikDrqbbY0Pu-9SP) - Nathen Harvey

Comment (eshellman) [reply courses/free-courses-en.md:302]: just put your submission under devops

**Pull request merged**